src/reducers/index.js

```diff
@@ -0,0 +1,28 @@
+
+export default (state = [], action) => {
+  switch (action.type) {
+    case 'ADD_ITEM':
```

Comment:
Please use the action type you defined before https://github.com/qq7886/react-redux-homework/blob/dev%2Fredux-table/src/actions/index.js#L2-L4

```js
import { ADD_ITEM, DELETE_ITEM, SAVE_ITEM } from './xxx/actions/xxx'
```